Formula/capnp.rb

```diff
@@ -26,9 +26,9 @@ def install
     gtest = resource("gtest")
     gtest.verify_download_integrity(gtest.fetch)
     inreplace "src/CMakeLists.txt" do |s|
-      s.gsub! "http://googletest.googlecode.com/files/gtest-1.7.0.zip",
+      s.gsub! "https://github.com/google/googletest/archive/release-1.7.0.zip",
```

Comment:
Unsure, but upstream hardcode in 1.7.0 & I presume that's what they expect other people to test against for now 😓.